inst/extdata/installDependencies.R

```diff
@@ -79,7 +79,16 @@ InstallPackages(pkgs, "http://mran.microsoft.com/snapshot/2016-03-31")
 
 # packages that must be held back at older versions
 InstallPackages("httr", "http://mran.microsoft.com/snapshot/2016-01-27")
-InstallPackages("devtools", "http://mran.microsoft.com/snapshot/2015-03-13")
+
```

Comment:
The R script was initially a line-by-line translation of the shell script, so honest mistake! :)